db/migrate/20190221181516_create_question_inputs.rb

```diff
@@ -0,0 +1,14 @@
+class CreateQuestionInputs < ActiveRecord::Migration[5.2]
+  def change
+    create_table :question_inputs do |t|
+      t.references :question, foreign_key: true
+      t.integer :allow_multiple_selections, default: 0
+      t.text :text_numeric, default: nil
```

Comment:
`text_numeric` and `text_alphanumeric` are not used to store input from the user, the purpose is instead to define which input types are supported for a question sent from the server to the user.

I checked the code and we only check if the value is null or not when we build the input questions. Thus it does not really matter of which type they are but to be consistent and save a few bytes we shall change them back to integers.

Reply:
I reverted the type changes, so now I just need a new approval for the pull request.

Reply:
You did not actually revert anything, instead you change another column from integer to text 😄 

So now you have two reverts to do 😄 In this table, fields `text_numeric`, `text_alphanumeric` and `audio` should be of type integer. Not that it really matters be we save a few bytes and that's an optimisation that is better to do know than later.

Reply:
Hahah, that's what happens when you're unfocused. I'll fix it in a bit.